content/headless/chunk/option-destination-diskbuffer.md

```diff
@@ -87,6 +96,27 @@ If the `dir()` path provided by the user does not exist, {{% param "product.ose"
 
 Options `reliable()` and `disk-buf-size()` are required options.
 
+### truncate-size-ratio() {#diskbuf-trunkate-size-ratio}
+
+| Type:        | number((between 0 and 1))    |
+|--------------|-----------|
+| Default:     | 0.1 (10%)   |
+
+*Description:* Limits the truncation of the disk-buffer file. Truncating the disk-buffer file can slow down the disk IO operations, but it saves disk space, so {{% param "product.abbrev" %}} only truncates the file, if the possible disk gain is more than `truncate-size-ratio()` times `disk-buf-size()`.
+
+By default, {{% param "product.abbrev" %}} frees the disk-space only when it can free up at least 10% of the disk-buffer file (`truncate-size-ratio(0.1)`):
```

Comment:
Related to my other comment, by default on `4.x` config versions, the disk-buffer file is never truncated.